bitnami/grafana-tempo/templates/vulture/servicemonitor.yaml

```diff
@@ -8,19 +8,9 @@ apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   name: {{ template "grafana-tempo.vulture.fullname" . }}
-  {{- if .Values.metrics.serviceMonitor.namespace }}
-  namespace: {{ .Values.metrics.serviceMonitor.namespace }}
-  {{- else }}
-  namespace: {{ .Release.Namespace }}
-  {{- end }}
```

Comment:
You have accidentally removed this piece of code in the new changes.

Reply:
good catch! Fixed at https://github.com/bitnami/charts/pull/18509/commits/2b1bf76f382de88a24f09b272be5503a673ca1fe